.gitignore

```diff
@@ -56,4 +56,5 @@ test/endToEnd/util/effectsTableDisplayTestUtil.js
 test/endToEnd/util/ITestCriterionRow.js
 test/endToEnd/util/ITestCriterionRowWithUncertainty.js
 test/endToEnd/workspaceSortingTest.js
-test/endToEnd/onlyEmptyAndTextTest.js
\ No newline at end of file
+test/endToEnd/onlyEmptyAndTextTest.js
+test/endToEnd/stepSizeTest.js
```

Comment:
https://trello.com/c/CD5vHmDl/3544-mcda-make-end-to-end-tests-no-longer-compile-in-place